File: Natural.c
#include <stdio.h>
void main() {
  int Count, Counter=1;
  printf("How many natural numbers you want to print?");
  scanf("%d", &Count);
  printf("The natural numbers upto %d are",Count);
  while(Counter <= Count) {
    printf(" %d", Counter);
    Counter = Counter + 1;
  }
  printf(".");
  printf("\n");
}

File: javaprogram/Natural.java
import java.lang.*;
import java.util.*;
class Natural {
  public static void main(String[] args) {
    Scanner ScannerObject = new Scanner(System.in);
    System.out.print("How many number of natural numbers you want to print?");
    int Count = ScannerObject.nextInt();
    int Counter = 1;
    System.out.print("The first " + Count + " natural numbers are");
    while(Counter <= Count) {
      System.out.print(" "+ Counter);
      Counter = Counter + 1;
    }
    System.out.print(".");
    System.out.println("");
  }
}